source/includes/fact-5.0-non-transactional-config-reads.rst

```diff
@@ -0,0 +1,10 @@
+Starting in MongoDB 5.0, non-transaction reads are not allowed on
+:ref:`config.transactions<config.transactions>` 
```

Comment:
Two things:

1) Please use
```
:data:`config.transactions`
```
for this link instead of a `ref` link.

2) I think we should say:
```
not allowed on the :data:`config.transactions` collection with the following...
```